addons/web/static/src/search/control_panel/control_panel.xml

```diff
@@ -41,9 +41,9 @@
 
             <div class="o_cp_pager"/>
 
-            <t t-if="props.viewSwitcherEntries and props.viewSwitcherEntries.length > 1">
+            <t t-if="viewSwitcherEntries.length">
```

Comment:
does it always exist? Seems weird for client actions

Reply:
Yes, it's always set by the SearchModel